src/utils.js

```diff
@@ -193,7 +203,8 @@ module.exports = {
     authenticateRequest,
     axiosGet,
     errorLogger,
-    isDomainBlacklisted,
+    isBlacklisted,
+    resolveDomain,
```

Comment:
Minor nitpick; can we keep alphabetical ordering? Thanks! :)

Reply:
Fixed :)